### DISCUSSION TITLE
Testcase1 error (HDF4)

### DISCUSSION THREAD

**Discussion by tais958** [NASA-LIS/LISF#1059] [Q&A] [answered]
Hi,

Glad now I am able to really work on the testcase. I ran into an issue that, HDF4 seems to be required in the first testcase which mainly uses LDT to generate parameter file. I recall the HDF4 is optional when compiling, which is not consistent with this. Any comments? Should I try to compile again with HDF4 support?

Thanks.

**Comment by tais958** [comment, marked as answer]
Just to update that I was able to install HDF4 and run through testcase 1 to generate the parameter file. Please still comment below if you have any. Thank you!

**Reply by bmcandr** [reply]
Hi @tais958, yes the public testcase currently requires that the LISF components be built with HDF4 and HDF5 support. I have opened #1053 about adding a note about this to our testcase walkthrough.

Apologies for the inconvenience!

**Comment by Lise-Li** [comment]
Hi, @tais958 .I met the same problem with you. Have you compiled HDF4 successfully? I met this problem as follows when I compiled the HDF4.Would you mind letting me know how you configured HDF4? Thank you.
mfgrff.f:155:48:

  132 |       mgsnatt = mgisattr(riid, name, nt, count, data, len(name))
      |                                                2
......
  155 |       mgsattr = mgisattr(riid, name, nt, count, data, len(name))
      |                                                1
Error: Type mismatch between actual argument at (1) and actual argument at (2) (CHARACTER(*)/INTEGER(4)).
